yrs/src/block.rs

```diff
@@ -765,6 +894,8 @@ impl ItemContent {
         }
     }
 
+    /// Checks if item content can be considered countable. Countable elements contribute to
+    /// a length of the block they are contained by.
```

Comment:
The block length is not affected by is_countable. Describe the difference between block-length and length of the shared type